pkg/kds/store/sync.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/go-logr/logr"
 
+	"github.com/kumahq/kuma/pkg/core"
```

Comment:
I'd love to avoid depending on this global time. It makes unit tests harder. We should have the clock passed as a param

Reply:
You can also usually write the tests in such a way as to not depend on the time itself